src/Apps/ViewingRoom/Components/ViewingRoomHeader.tsx

```diff
@@ -39,6 +39,53 @@ export const ViewingRoomHeaderFragmentContainer = createFragmentContainer(
   }
 )
 
+/**
+ * Header used for desktop layouts
+ */
+const ViewingRoomHeaderLarge: React.FC<ViewingRoomHeaderProps> = props => {
+  const {
+    viewingRoom: { heroImageURL, title },
+  } = props
+
+  return (
+    <Flex
+      style={{
+        height: `calc(100vh - ${NavBarHeight}px)`,
```

Comment:
I never trust this sort of thing — this is just because we can't flex: 1 out the parent?

Reply:
Can you explain a bit more? I can try it, but my css is also really bad 😄

Reply:
Like: https://codepen.io/dzuc/pen/OJyEbzN

Vs trying to calculate the height of of the header then subtract it out.